### DISCUSSION TITLE
Website: Game pages Text

### DISCUSSION THREAD

**Discussion by kyleofawesome** [gazzabwoy/projectEprom#24] [General]
Heres all the text for the game pages i couldent figure out to make a seprate brach so here. 

[Game Text.zip](https://github.com/gazzabwoy/projectEprom/files/7728494/Game.Text.zip)
.

**Comment by gazzabwoy** [comment]
Well done kyle, I have now updated the code..

![image](https://user-images.githubusercontent.com/91916706/146409966-05cee3fd-737c-4303-a474-63b01f697759.png)

@fagann , just need you to finish the front page and the site is complete. Can you manage it?

**Comment by gazzabwoy** [comment]
![image](https://user-images.githubusercontent.com/91916706/146410478-4a0e59bb-45b1-4d05-b940-bcfb0ea27e56.png)

This is the start of my new project farm for Ryan's class, doing a different sorta menu system.
Site is inspired by Diddly Squat Farm Shop Website ( Jeromy Clarkson's Farm)

**Comment by fagann** [comment]
Hi Gary, i am so sorry that i couldn't respond to your emails as i never check them. Because sometimes i might forget to check. Today i was going through my emails and saw yours. Again i am sorry 😄
________________________________
